### PR TITLE
kubelet/rkt: bump rkt version for rkt image list fix.

### PR DESCRIPTION
**Pull request by yifan-gu**
This PR bumps the rkt version to v0.8.1.
Also as https://github.com/coreos/rkt/pull/1388 gets merged, this updates the `rkt image list` command.

/cc @dchen1107 @yujuhong

### CODE REVIEW

File: pkg/kubelet/rkt/rkt.go [hunk 1]
```diff
@@ -943,7 +943,7 @@ func (r *runtime) PullImage(image kubecontainer.ImageSpec, pullSecrets []api.Sec
 func (r *runtime) IsImagePresent(image kubecontainer.ImageSpec) (bool, error) {
 	repoToPull, tag := parseImageName(image.Image)
 	// TODO(yifan): Change appname to imagename. See https://github.com/coreos/rkt/issues/1295.
-	output, err := r.runCommand("image", "list", "--fields=appname", "--no-legend")
+	output, err := r.runCommand("image", "list", "--fields=name", "--no-legend")
```
Comment (dchen1107) [inline pkg/kubelet/rkt/rkt.go]: Give me an example of output from running the command here?

Comment (dchen1107) [inline pkg/kubelet/rkt/rkt.go]: I mean as comment

File: pkg/kubelet/rkt/rkt.go [hunk 2]
```diff
@@ -1277,7 +1277,7 @@ func (r *runtime) getImageByName(imageName string) (*kubecontainer.Image, error)
 
 // ListImages lists all the available appc images on the machine by invoking 'rkt image list'.
 func (r *runtime) ListImages() ([]kubecontainer.Image, error) {
-	output, err := r.runCommand("image", "list", "--no-legend=true", "--fields=key,appname")
+	output, err := r.runCommand("image", "list", "--no-legend=true", "--fields=key,name")
```
Comment (dchen1107) [inline pkg/kubelet/rkt/rkt.go]: ditto

**Comment by dchen1107** [comment]
Besides those two nits, LGTM

**Comment by yifan-gu** [comment]
@dchen1107 Thanks :) Updated.

**Comment by dchen1107** [comment]
LGTM

**Comment by alex-mohr** [comment]
@yifan-gu fyi, this PR blocked due to shippable still failing.

**Comment by yujuhong** [comment]
Shippable failure seems like a flake. Reopened the PR to kick off another run.

**Pull request merged**